scripts/apps/templates/directives/TemplatesDirective.ts

```diff
@@ -348,6 +364,10 @@ export function TemplatesDirective(notify, api, templates, modal, desks, weekday
                 {label: gettext('No Desk'), value: 'None'},
             ];
 
+            if (templates.isAdmin(true)) {
+                $scope.filters.splice(2, 0, {label: gettext('Private'), value: 'Private'});
```

Comment:
Can you use `push` instead? It's easier to follow the logic when it's sequential without having to wonder what makes `2` the special position.

Reply:
Done

Reply:
The splice is still here. Can you use `push` for `{label: gettext('Private'), value: 'Private'}` too?

Reply:
oops that should have gone! Private gets pushed conditionally on line 367